### PR TITLE
LPS-147366 fixed menu and logo

### PR DESCRIPTION
**Pull request by Mylena-Rodrigues**

### CODE REVIEW

**Comment by Mylena-Rodrigues** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/quote-navbar/index.html [hunk 1]
```diff
@@ -23,11 +23,11 @@
...
 		</div>
 
-		<div class="menu-drop">
-			<button aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation" class="ghost navbar-toggler raylife-navbar-button" data-target="#navbarSupportedContent" data-toggle="collapse" type="button">
+		<div class="d-lg-none d-md-none d-sm-block menu-drop">
+			<button aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation" class="ghost navbar-toggler raylife-navbar-button raylife-quote-menu" data-target="#navbarSupportedContent" data-toggle="collapse" type="button">
 				<div class="align-items-center d-flex justify-content-between">
-					<span class="font-weight-bolder mr-1 paragraph-sm">MENU</span>
-					<span class="background-icon-grid" id="button-menu-icon"></span>
+					<span class="raylife-quote-menu font-weight-bolder mr-1 paragraph-sm">MENU</span>
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/quote-navbar/index.html:29]: Could you sort the class names?

File: modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/quote-navbar/index.js [hunk 2]
```diff
@@ -21,6 +21,25 @@ const menuIcon = fragmentElement.querySelector('#button-menu-icon');
...
+	'selected-quote',
+	'get-in-touch',
+];
+const logoElement = document.querySelector('.logo-desktop');
+const pathName = window.location.pathname;
+const menuElements = document.querySelectorAll('.raylife-quote-menu');
+
+pages.forEach((page) => {
+	if (pathName.includes(page)) {
+		menuElements.forEach((element) =>
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/quote-navbar/index.js:36]: Could you use menuElement instead element?

**Review by silasbsi — CHANGES_REQUESTED** [review]
LGTM :fire: , but needs some adjustments

**Review by silasbsi — APPROVED** [review]

**Comment by silasbsi** [comment]
LGTM :fire: @kevenleone could you make a new review?

**Comment by silasbsi** [comment]
ci:test:sf

**Review by kevenleone — APPROVED** [review]
LGTM 🔥 

Well done, @Mylena-Rodrigues

**Comment by kevenleone** [comment]
ci:forward

**Comment by silasbsi** [comment]
ci:forward

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**